pom.xml

```diff
@@ -55,12 +55,6 @@
     </dependency>
 
     <dependency>
-        <groupId>org.slf4j</groupId>
```

Comment:
we only need this OR slf4j-simple, having both was giving us some compile time warnings that were ignored